### PR TITLE
Update Trakx.Utils to 0.1.10

### LINKED ISSUE #8: Reference Trakx.Utils in Coinbase Client and get rid of duplicate implementations
Notably in the Utils folder

### PR DESCRIPTION
**Pull request by AymericNoel**
close #8

### CODE REVIEW

File: src/Trakx.Coinbase.Custody.ApiClient/CoinbaseCustodyApiConfiguration.cs
Comment (monsieurleberre) [inline src/Trakx.Coinbase.Custody.ApiClient/CoinbaseCustodyApiConfiguration.cs]: the ones which are not secret should simply be shared in the appsettings when you commit
this is just to indicate what we consider secret, and to generate readme doc automatically for other devs to get the .env file ready rapidly.
also cf other PR comments, I think 0.1.11 lets you not add the variable name

File: src/Trakx.Coinbase.Custody.ApiClient.Tests/Secrets.cs
Comment (monsieurleberre) [inline src/Trakx.Coinbase.Custody.ApiClient.Tests/Secrets.cs]: let's see if using SecretProvider<T> makes sense here.

**Review by monsieurleberre — COMMENTED** [review]
👍

**Comment by AymericNoel** [comment]
Can't merge without approvals

**Review by monsieurleberre — APPROVED** [review]
sorry I meant to approve in the first place

**Pull request merged**
